bin/main.ml

```diff
@@ -7,12 +7,36 @@ open Fiber.O
    bootstrap, so we set this reference here *)
 let () = suggest_function := Cmdliner_suggest.value
 
+module Arg = struct
+  include Arg
+
+  let package_name =
+    Arg.conv ((fun p -> Ok (Package.Name.of_string p)), Package.Name.pp)
+
+  module Path : sig
+    type t
+    val path : t -> Path.t
+    val arg : t -> string
+
+    val conv : t conv
+  end = struct
+    type t = string
+
+    let path p = Path.of_filename_relative_to_initial_cwd p
+    let arg s = s
+
+    let conv = Arg.conv ((fun p -> Ok p), Format.pp_print_string)
+  end
+
+  let path = Path.conv
```

Comment:
Can we have `let file = path` as well? Just to make sure we don't use the wrong thing

Reply:
Done